GOVERNANCE.md

```diff
@@ -65,6 +65,23 @@ meeting. In both cases the decision must be seconded by a majority of the Team
 or else the discussion will continue. Simple majority wins. Only Active
 Members participate in a vote.
 
+## Aproving Project Charters
+
+Per the [CPC charter section 5](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc), the voting CPC members are responsible for approving project
+charters and changes to them.
+
+Approval of the initial charter or changes to an existing charter will
+be requested by opening an issue requesting approval in the cross-project-council
+[repository](https://github.com/openjs-foundation/cross-project-council/)
+
+The request is approved when:
+conditions:
+
+* there are no outstanding objections
+* there are two or more approvals by voting CPC members
```

Comment:
I have mixed feelings about this tbqh. On one hand this will be good in combination with the 14 day rule.. that being said if we have a meeting with quorum of CPC voting members and no objections should it have to wait?

Reply:
I still come back to "how do you know there are no objections".  Since potentially half the of the members have not seen the PR you don't know.  Yes the 1/2+1 can always override the other 1/2-1 but since we are consensus seeking we should be looking to address objects if there are any, not just ignore them because we have 1/2+1 who agree.  

In my opinion, if in the end if we just say that 1/2+1 can make the decision without giving the rest of the members the option to object and looking to address those objections, we are not really following consensus seeking.

Reply:
I'll also add that it's ok if we think that some decisions should be majority rule versus consensus seeking but I think that should be explicitly stated that we've made that decision versus it being consensus seeking unless 1/2+1 show up at a meeting and agree.

Reply:
After reading through all the text in context I think I'm okay with it. If we are talking about shortening the time then a solution like a super-majority might be useful, but here it seems like we're trying to address the situation where apathy or the lack of controversy has caused people to ignore the issues. I think we owe it to projects to give them autonomy over their charters unless there are objections within a reasonable period.